features/provider/admin/authentication_providers/edit.feature

```diff
@@ -0,0 +1,33 @@
+@javascript
+Feature: Audience > Developer Portal > Settings > SSO Integrations > Edit
+
+  Background:
+    Given a provider is logged in
+    And the provider has "branding" switch allowed
+    And the provider has "iam_tools" switch allowed
```

Comment:
Are these required?

Reply:
Yes. This is because by default (and also in tests) the settings have `onpremises: false`, so "SaaS mode" is applied, and the new providers have these switches disabled, so they don't have access to SSO integrations.

Reply:
Understood 👍